liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/PackageServerMojo.java

```diff
@@ -89,10 +77,12 @@ protected void doExecute() throws Exception {
         ServerTask serverTask = initializeJava();
         copyConfigFiles();
         serverTask.setOperation("package");
-        String fileType = minificationType(include);
+        String fileType = getPackageFileType(include);
+        String projectBuildDir = project.getBuild().getDirectory();
+        String projectBuildName = project.getBuild().getFinalName();
         if (packageFile != null) {
             if (packageFile.isDirectory()) {
-                packageFile = new File(packageFile, serverName + fileType);
+                packageFile = new File(packageFile, projectBuildName + fileType);
```

Comment:
if packageFile is a directory, do we need to check to see if it ends with a / and add the / if not or is that already taken care of?  i.e., was this an existing bug?